### PR TITLE
Propose Lagoon %real release.

### PR DESCRIPTION
**Pull request by sigilante**

### CODE REVIEW

File: UIPS/UIP-0122.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+  $:  =meta           ::  descriptor
+      data=@ux        ::  data, row-major order
+  ==
+::
++$  prec  [a=@ b=@]   ::  fixed-point precision, a+b+1=bloq
++$  meta              ::  $meta:  metadata for a $ray
+  $:  shape=(list @)  ::  list of dimension lengths
+      =bloq           ::  logarithm of bitwidth
+      =kind           ::  name of data type
+      fxp=(unit prec) ::  fixed-point scale
```
Comment (mopfel-winrux) [inline UIPS/UIP-0122.md:43]: do we want to comment out fxp?

Comment (sigilante) [reply UIPS/UIP-0122.md:43]: How finicky are state upgrades going to be with adding a value here, I guess is the question.

Comment (sigilante) [reply UIPS/UIP-0122.md:43]: The real PITA is that we will have to `s///g` all the test cases.

File: UIPS/UIP-0122.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+
+In the long run, as more possibilities for userspace jets become available, `/lib/lagoon` may be moved out of `%base`.  However, one change in policy at a time is all that we propose now.
+
+### Rationale for `%real`
+
+IEEE 754 `%real`s are a well-understood, well-vetted component in the numerical ecosystem for scientific computing and Urbit's `@r` affordances.
+
+We have comprehensive unit test coverage of `/lib/lagoon` and SoftBLAS, lending credence to verified behavior.
+
+Taken together, these lead us to believe that introducing `%real` to the developer community first is a safe and
```
[the remark this answers is not in the document]
Comment (sigilante) [reply UIPS/UIP-0122.md:180]: “You're killing me, Smalls!”

**Review by belisarius222 — APPROVED** [review]

**Pull request merged**